[2018-10-04 01:00:52,036][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:23:22,473][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:23:27,421][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:23:33,050][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:23:38,060][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:23:38,710][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:24:19,704][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:24:29,861][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:24:33,011][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:25:03,700][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:25:37,351][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:27:01,532][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:27:12,136][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 01:28:27,282][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:38:43,278][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:39:00,806][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:40:11,800][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:40:43,707][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:51:40,634][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:53:37,461][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 05:54:03,390][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 06:10:03,277][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 06:11:37,878][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 06:12:20,497][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 06:12:38,698][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 06:14:35,011][ERROR][assign_task.py:30] In API calling assign_task_query_response
[2018-10-04 06:14:57,320][ERROR][assign_task.py:30] In API calling assign_task_query_response
